src/pykiso/logging_initializer.py

```diff
@@ -234,3 +239,94 @@ def initialize_loggers(loggers: Optional[List[str]]) -> None:
     loggers_to_deactivate = set(relevant_loggers) - set(active_loggers)
     for logger_name in loggers_to_deactivate:
         logging.getLogger(logger_name).setLevel(logging.WARNING)
+
+
+def import_object(path: str) -> Any:
+    """return the object based on the path.
+        For example : logging.Logger will return Logger
+
+    :param path: path to the object
+    """
+    if path:
+        components = path.split(".")
+        mod = importlib.import_module(components[0])
+        for comp in components[1:]:
+            mod = getattr(mod, comp)
+        return mod
+    else:
+        return None
+
+
+def add_filter_to_handler(func):
+    """Decorator function that will add a filter to all the handlers
+        of a logger after a function.
+
+    :param func: function to execute
+    """
+
+    def wrapper(self, *arg, **kwargs):
+        func(self, *arg, **kwargs)
+        for handler in self.handlers:
+            handler.addFilter(InternalLogsFilter())
+
+    return wrapper
+
+
+def remove_handler_from_logger(func):
+    """Decorator that will remove all handlers of a logger after
+        executing a function.
+
+    :param func: function to execute
+    """
+
+    def wrapper(self, *arg, **kwargs):
+        func(self, *arg, **kwargs)
+        for handler in self.handlers:
+            self.removeHandler(handler)
+
+    return wrapper
+
+
+def change_logger_class(log_level: str, verbose: bool, logger: str):
+    """Change the class of all the logger of pykiso.
+
+    :param log_level: level of the log
+    :param logger: str of the path to the logger class
+    """
+    # Get the argument to initialize the logger if needed
+    kwargs_log = {}
+    if "(" in logger:
+        logger_class, arg_logger = logger.split("(")
+        arg_logger = arg_logger[:-1].split(",")
+        kwargs_log = {arg.split("=")[0]: arg.split("=")[1] for arg in arg_logger}
+        # We try to load item to handle int, bool argument
+        for name, arg in kwargs_log.items():
+            try:
+                kwargs_log[name] = json.loads(arg.lower())
```

Comment:
It could be wiser to load the entire dict directly and raise if it fails instead of silently dropping the argument

Reply:
`json.loads` does not support dict, and the goal here is to transform the str given by the user to the appropriate type : 
'12' -> 12 
So if it fails it means it is just a str. The argument is not dropped he keeps the same value as given by the user.

Reply:
You could then consider using `ast.literal_eval` instead, which will allow strings passed as `my_str_param='str'` and `my_str_param="str"` as well as any other builtin python type

Reply:
Done